spec/models/comment_spec.rb

```diff
@@ -1,27 +0,0 @@
-require 'rails_helper'
-
-RSpec.describe Comment, type: :model do
-  subject { Comment.new(posts_id: '1', author_id: '3', text: 'Hi Selma!') }
-
-  before { subject.save }
-
-  it 'author id should be present' do
-    subject.author_id = nil
-    expect(subject).to_not be_valid
-  end
-
-  it 'post id should be present' do
-    subject.posts_id = nil
-    expect(subject).to_not be_valid
-  end
-
-  it 'text should be a string ' do
-    subject.text = 123
-    expect(subject).to_not be_valid
-  end
-
-  it 'text should be present' do
-    subject.text = nil
-    expect(subject).to_not be_valid
-  end
```

Comment:
- [ ] Kindly make sure to wrap all your tests in a `describe` statement, so your tests are grouped
- [ ] Also make sure to Create Unit tests for your `Comment` model, they got removed after you revert changes to this commit